tsconfig.json

```diff
@@ -1,7 +1,11 @@
 {
     "files": [],
-    "references": [
-        { "path": "server" },
-        { "path": "example" }
+    "extends": "./tsconfig.settings.json",
```

Comment:
I'd just merge those files now since there is no need to reuse the settings in two places anymore.